src/js/pinny.js

```diff
@@ -431,6 +431,10 @@
                         }.bind(this)
                     )
                     .on(events.blur, this._hideSpacer.bind(this));
+                // Issue 73: We should perform these actions in
+                //  addition to binding them
+                this._showSpacer();
```

Comment:
So rather than repeating these lines of code, you could chain a call to `trigger(events.focus)` after line 432.